lib/statsample-glm/glm/formula.rb

```diff
@@ -29,9 +29,11 @@ def parse_formula(form = :token)
 
       private
 
-      def add_contant_term_if_required
-        # TODO: Add support for constants
-        @tokens.unshift Token.new('1')
+      def manage_contant_term
+        @tokens.unshift Token.new('1') unless
+          @tokens.include?(Token.new('1')) ||
+          @tokens.include?(Token.new('0'))
+        @tokens.delete Token.new('0')
```

Comment:
Try to make it class method of `Statsample::GLM`, and feel free to ask about problems/errors, let's think about it together (there are several not-so-intuitive things in name resolutions in similar cases, so I generally prefer try-and-fail-and-try-again approach, it is hard to invent "definitely working solution" from the first attempt)